src/wwwroot/directives/white-brand-toggler-text.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+    'use strict';
+
+    angular
+      .module('dopplerRelay')
+      .directive('whiteBrandTogglerText', whiteBrandTogglerText);
+
+      whiteBrandTogglerText.$inject = [
+        'utils',
+        '$window',
+        '$rootScope'
+    ];
+
+    function whiteBrandTogglerText(utils, $window, $rootScope) {
+        var directive = {
+            restrict: 'A',
+            link: function (scope, element, attrs, ctrls) {
+              var hasDifferentDomain = utils.hasDifferentDomain();
+              if(hasDifferentDomain) {
+                var usersWithMultiAccount = utils.getMultiAccountUsers();
+                var userWithMultiAccount = usersWithMultiAccount.reduce(function(user) {
+                  var regex = new RegExp('\\b' + user.domain + '\\b');
+                  if (regex.test($window.location.hostname)) {
+                    return { companyName: user.companyName, logoName: user.logoName, domain: user.domain, companyLink: user.companyLink }
+                  };
+                });                
+                if (!!userWithMultiAccount && element.parent().parent()[0].tagName == 'FOOTER') {
```

Comment:
It sounds like an ugly patch, I mean, it does not look very generic. I understand it, and I think that we can live with it, but could you add some comments in the code when you had to apply this kind of patches?

Reply:
Sure, i'm currently trying to do something about that :(

Reply:
No problem, I understand, what do you think about simply adding a comment describing why is this patch here?